### DISCUSSION TITLE
Getting ValueError: Invalid RGBA argument: 'ffd12e', when plotting tyre compound colours

### DISCUSSION THREAD

**Discussion by 3ric03** [theOehrly/Fast-F1#378] [Q&A]
Hi, I'm following the following code:
https://docs.fastf1.dev/examples_gallery/plot_driver_laptimes.html#sphx-glr-examples-gallery-plot-driver-laptimes-py
Which plots a driver's laptimes throughout a race in a scatter plot, when I try to compile the example code, I get the error ValueError: Invalid RGBA argument: 'ffd12e'.
I think the error is generated from the line hue="Compound", which seems to not pass in the correct RGB values?
Any help will be appreciated, thank you in advance

**Comment by theOehrly** [comment]
> Hi, I'm following the following code: https://docs.fastf1.dev/examples_gallery/plot_driver_laptimes.html#sphx-glr-examples-gallery-plot-driver-laptimes-py Which plots a driver's laptimes throughout a race in a scatter plot, when I try to compile the example code, I get the error ValueError: Invalid RGBA argument: 'ffd12e'. I think the error is generated from the line hue="Compound", which seems to not pass in the correct RGB values? Any help will be appreciated, thank you in advance

This issue should be fixed in the newest release of FastF1. Please update and report back if everything is working correctly then.

**Reply by 3ric03** [reply]
Thank you for your response, how can I install the newest release using pip?

**Reply by theOehrly** [reply]
``pip install --upgrade fastf1``
